.circleci/config.yml

```diff
@@ -231,11 +232,11 @@ jobs:
       - attach_workspace:
           at: ~/player
 
-      - run: bazel test --config=ci -- //third_party/hermes/hello:test
+      - run: bazel build --config=ci -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
```

Comment:
Hmm, I'm not understanding the change here, why are we building more things at this stage that aren't related to what this specific run step was for (just building `@hermes//:host` in isolation). If that's no longer required, can we just remove this step?

Reply:
This and your other comment about the filters are related to the same theoretical change. In essence what this does is same same thing as the existing `bazel test --config=ci -- //third_party/hermes/hello:test` invocation which is to build hermes without any potential flags which may cause bazel to invalidate its cached build. When both test invocations were the same minus setting the android runtime flag it was causing Hermes to rebuild on the subsequent run. This also splits up the actual building of the tests and their execution just to be extra safe around how much is actively running. There is probably a better way to do this though so I am open to suggestions.